### ISSUE TITLE
CMT numbers for EML project

### ISSUE THREAD

**Issue by tobeycarman** [ua-snap/dvm-dos-tem#315]
@hgenet, can you verify that CMTs 8, 9 and 10 [here](https://github.com/ua-snap/dvm-dos-tem/tree/joys-calibrations/parameters) should be moved to the EML cmt number space (20-39)?

**Comment by hgenet** [comment]
Tobey,

CMT9 and 10 are EML/DeepRoot. But CMT 8 (shrubland) should be listed in 
the IEM group. Because it is not a CMT identified in the land cover map 
yet, we should give it a number >= 12 (landcover classes in the 
landcover maps go from 0 to 11). This Shrubland parameterization will be 
used down the line when ATM and DVMDOSTEM will be coupled.

Helene

----
Hélène Genet, PhD
Institute of Arctic Biology
University of Alaska Fairbanks
902 Koyukuk Drive
Fairbanks, AK 99775 - USA
Phone: +1 907 699 4340
Email: hgenet@alaska.edu
Skype: helenegenet

**Comment by tobeycarman** [comment]
OK, sounds good. Sorry, I know you mentioned that the other day, I should have remembered. 

Is the name "Shrubland" appropriately distinct from "Shrub Tundra"?

FWIW, the veg map I have been using (from `atlas:/atlas_scratch/tem/snap-data/iem_ancillary_data/Landcover/LandCover_iem_TEM_2005.tif`) only has classes 0-7:

![screen shot 2017-10-02 at 1 37 14 pm](https://user-images.githubusercontent.com/838735/31100726-12402404-a777-11e7-9bc3-88253e0858a3.png)

**Comment by hgenet** [comment]
Yes Tobey, Shrubland is different from Shrub tundra. This shurbland 
parameterization has been developed from sites selected and described by 
Vijay Patil in the Yukon Flats for his PhD. One way to describe them 
would be lowland boreal wet shrubland.

Helene

----
Hélène Genet, PhD
Institute of Arctic Biology
University of Alaska Fairbanks
902 Koyukuk Drive
Fairbanks, AK 99775 - USA
Phone: +1 907 699 4340
Email: hgenet@alaska.edu
Skype: helenegenet

**Comment by tobeycarman** [comment]
OK, maybe I will put it as "Lowland Boreal Wet Shrubland". Kind of a mouthful, but it is very explicit, which I usually appreciate down the line.

Also note that there are some options for the `param-util.py` script that make it easy to check that all the files are consistent. `dmvdostem` itself doesn't really pay heed to the comments for the CMTs in the parameter files, but some of our downstream programs and scripts (calibration viewers etc) will can be enhanced with nice text labels if the names in our files are consistent. And our integration with PEcAn will also benefit from keeping things nice clean and standard. 

Here is an example of how I have been using `param-util.py` to see that the community types are named/numbered the same across different files (this is **before** I edited as we have talked about above):

```
$ ./scripts/param_util.py --report-cmt-names parameters/ 8
                                    file name  cmt key   long name
                   parameters/cmt_bgcsoil.txt    CMT08   CMT8 Shrubland Community
             parameters/cmt_bgcvegetation.txt    CMT08   Shrubland
                 parameters/cmt_calparbgc.txt    CMT08   Shrubland
                 parameters/cmt_dimground.txt    CMT08   Shrubland
             parameters/cmt_dimvegetation.txt    CMT08   Shrubland
                 parameters/cmt_envcanopy.txt    CMT08   CMT8 Shrubland Community
                 parameters/cmt_envground.txt    CMT08   CMT8 shrubland TEM Community
                   parameters/cmt_firepar.txt    CMT08   CMT8 shrubland Community
```
